app/cluster_discovery.go

```diff
@@ -16,38 +16,42 @@ const (
 
 type ClusterDiscoveryService struct {
 	model.ClusterDiscovery
-	app  *App
-	stop chan bool
+	server *Server
```

Comment:
Would suggest it to be `srv` to maintain consistency with the naming.

Reply:
:+1: